var/spack/repos/builtin/packages/python/package.py

```diff
@@ -208,6 +208,8 @@ def post_install(self):
         spec = self.spec
         prefix = self.prefix
 
+        install(os.path.join(self.package_dir, 'sitecustomize.py'), os.path.join(prefix.lib, 'python2.7', 'site-packages'))
```

Comment:
this will only work for `python@2.7`, try something like `self.spec.version.up_to(2)`

Reply:
I added a TODO comment.  I'm not sure what's going to happen with Python 3, so hardcoding up_to(2) doesn't seem right either....